cmd/utils/nodecmd/flags_test.go

```diff
@@ -472,7 +472,7 @@ var flagsWithValues = []struct {
 	{
 		flag:        "--exec",
 		flagType:    FlagTypeArgument,
-		values:      []string{"klay.blockNumber", "klat.getBlock(0)", "governance.chainConfig.governance.reward.proposerUpdateInterval"},
+		values:      []string{"klay.blockNumber", "klat.getBlock(0)", "governance.chainConfigAt().governance.reward.proposerUpdateInterval"},
```

Comment:
I think this need to be updated as well (and the PR title)

Reply:
Thank you! I have changed this file also.